connectors/sources/atlassian.py

```diff
@@ -186,7 +203,10 @@ def is_active_atlassian_user(self, user_info):
             )
             return False
 
-        if user_info.get("accountType") != "atlassian":
+        if (
+            self.source.configuration["data_source"] in [JIRA_CLOUD, CONFLUENCE_CLOUD]
+            and user_info.get("accountType") != "atlassian"
+        ):
             self.source._logger.debug(
                 f"Skipping {user_name} because the account type is {user_info.get('accountType')}. Only 'atlassian' account type is supported."
```

Comment:
Should this log message get updated too? Since now the account type could be `atlassian`, but the data source config is non-cloud.

Reply:
This is designed to be logged for just cloud data source according to this condition
```
self.source.configuration["data_source"] in [JIRA_CLOUD, CONFLUENCE_CLOUD]
```

Previously, DLS was designed to just cloud platform so did not have this condition but now we've explicitly added it just to exclude non-cloud data source.